src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+public enum ColorSupplier {
+    RED,
+    GREEN,
+    BLUE,
+    YELLOW,
+    ORANGE;
```

Comment:
move it to a separate file
color supplier should be in its own file
avoid static methods
make random an instance field (class-level), not a local variable, this is inefficient - we recreate random per each method call